protocol/src/main/java/org/opensearch/sql/protocol/response/format/FlatResponseFormatter.java

```diff
@@ -30,6 +30,10 @@ public FlatResponseFormatter(String seperator, boolean sanitize) {
     this.sanitize = sanitize;
   }
 
+  public String getFormat() {
```

Comment:
Could the `getFormat` functions in `protocol/src/main/java/org/opensearch/sql/protocol/response/format/FlatResponseFormatter.java` and `protocol/src/main/java/org/opensearch/sql/protocol/response/format/JsonResponseFormatter.java` be differentiated by name?



Perhaps, `getFormatPlainText` and `getFormatApplicationJson`?

Reply:
What would we gain from giving the functions different names when we want to call `getFormat` in `RestSQLQueryAction`?

Reply:
I will close this as the methods need to share the same name as they are inherited from the same abstract and need to be the same to be called in `RestSqlQueryAction`